cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java

```diff
@@ -54,8 +54,8 @@
 
     @Override
     public void execute(JobExecutionContext context) throws JobExecutionException {
-      LOG.debug("Trying to run job {} with trigger {}", context.getJobDetail().getKey().toString(),
-                context.getTrigger().getKey().toString());
+      LOG.debug("Emitting time notification for program '{}' and schedule '{}'.",
+                context.getJobDetail().getKey().toString(), context.getTrigger().getKey().toString());
       Trigger trigger = context.getTrigger();
```

Comment:
@anew this change is regarding your comment in the bug https://issues.cask.co/browse/CDAP-11534.